### ISSUE TITLE
Prompt before re-search

### ISSUE THREAD

**Issue by m-sundman** [qarmin/czkawka#661]
If you press the big Search button again (perhaps thinking it will allow searching among the existing results) an extremely lengthy search takes place, and cancelling it won't take you back to your existing results. If you've just waited for a 4 hour search to finish then you certainly won't be happy to suddenly have to wait for a very long time again just because the UI was unclear.

**Comment by alexislefebvre** [comment]
The results should have been cached, so it should be way faster on the second scan. Maybe you have an issue with the scan.

The question is still valid though.

**Comment by m-sundman** [comment]
Yes, the second scan is much faster, but that was still 15-20 minutes of dead time just because the UI is unclear. Imagine the same happening in some other product. Like, you press the "open doors" button in the elevator while it's moving and it would just freeze there for 20 minutes before it lets you continue on your way.

**Comment by m-sundman** [comment]
"Scan" (which would say "Scan again" if you're already viewing a scan result) is better than "Search" for sure, but I also think it would be good with a confirmation box asking if you're sure that you want to re-run the scan. (Possibly only if the original scan was very lengthy, but it probably wouldn't hurt to ask anyway, before you throw away any existing selections or make the user wait.)
